Assets/MRTK/Core/Utilities/Async/AwaiterExtensions.cs

```diff
@@ -141,12 +141,23 @@ private static void RunOnUnityScheduler(Action action)
         {
             if (SynchronizationContext.Current == SyncContextUtility.UnitySynchronizationContext)
             {
+                // Take the opportunity to ensure AsyncCoroutineRunner is running when we are on the Unity thread
+                if (!AsyncCoroutineRunner.IsInstanceRunning)
+                {
+                    var _ = AsyncCoroutineRunner.Instance;
+                }
```

Comment:
Do you think we need this? As far as I see, all the `action`s contain `AsyncCoroutineRunner.Instance`, which should ensure an instance in the scene, which will happen on line 149 when `action()` is called.